### DISCUSSION TITLE
Caused by Data Quality how to monitor

### DISCUSSION THREAD

**Discussion by linzhicong1996** [apache/doris#24188] [A - General / Q&A]
When I used flink  import data to doris, I found that flink would try to re-import due to data quality errors。i want to flink pass err data and send email to Monitor。but if i use max_filter_ratio=1 ，flink cant feedback err info to flink program.Is there any way to both skip and monitor error imports?

**Comment by yagagagaga** [comment]
You can check data quality before sink to Doris. For example, you can add a map operator and immediately throw an exception if the data quality does not meet expectations.
